packages/icons/package.json

```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "@babel/cli": "^7.2.3",
     "@svgr/cli": "^4.1.0",
+    "jest": "^24.1.0",
```

Comment:
not sure about this one

Reply:
I wasn't able to run the tests until I added this, since jest wasn't being installed

Reply:
I just did a local test. I am able to do `npm ci && npm t` and confirmed the pcln-icons tests being executed. Definitely don't need to add jest in pcln-icon to have test working.

```
→ npm t

> @ test /Users/bchen/github/design-system
> jest

 RUNS  packages/icons/test/index.js
```

Reply:
UPDATED . Thanks folks!